webapp/src/components/App.tsx

```diff
@@ -9,6 +9,7 @@ import Navbar from './Navbar';
 import ReflectionsPage from './ReflectionsPage';
 import RequireAuth from './RequireAuth';
 import SessionContext from './SessionContext';
+import CompetenciesPage from './CompetenciesPage';
```

Comment:
Should this be imported in alphabetical order?

Reply:
Yes, it should to follow convention.